binding/react/src/rhino_types.ts

```diff
@@ -106,20 +125,31 @@ export interface RhinoWorkerFactory {
   create: (rhinoArgs: RhinoArgs) => Promise<RhinoWorker>;
 }
 
+export type RhinoWorkerResponseFileOperation = {
+  command: 'file-save' | 'file-load' | 'file-exists' | 'file-delete';
+  path: string;
+  content?: string;
+};
+
 export type RhinoWorkerResponse =
   | RhinoWorkerResponseReady
   | RhinoWorkerResponseInference
   | RhinoWorkerResponseError
   | RhinoWorkerResponseInitError
-  | RhinoWorkerResponseInfo;
+  | RhinoWorkerResponseInfo
+  | RhinoWorkerResponseFileOperation;
 
 // React
 
 export type RhinoHookArgs = {
-  /** Immediately start the microphone upon initialization */
-  start: boolean;
+  /** AccessKey obtained from Picovoice Console (https://picovoice.ai/console/) */
+  accessKey: string;
   /** The context to instantiate */
   context: RhinoContext;
+  /** If set to `true`, Rhino requires an endpoint (chunk of silence) before finishing inference. **/
+  requireEndpoint?: boolean;
+  /** Immediately start the microphone upon initialization */
+  start: boolean;
```

Comment:
It's optional in angular but not here. why?

Reply:
Legacy oversight, fixed.